packages/react-components/react-nav-preview/src/components/NavItem/NavItem.types.ts

```diff
@@ -1,8 +1,9 @@
-import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { ARIAButtonSlotProps } from '@fluentui/react-aria';
```

Comment:
This one should be a type
```suggestion
import type { ARIAButtonSlotProps } from '@fluentui/react-aria';
```